### ISSUE TITLE
Framerate issue

### ISSUE THREAD

**Issue by NSAgent** [LukeRoss00/gta5-real-mod#273]
I'm wondering if I did something wrong. Framerate is extremely low, I've tried on both H and M graphics settings, same result.

My setup
GPU: GTX 1070 Ti
Ryzen 3700X
16 GB RAM
Oculus Quest 2
Not sure where to find the game version and build but I just bought and installed it from Steam, no mods whatsoever
Followed all steps, went over them twice.

**Comment by LukeRoss00** [comment]
You're most likely using an outdated version of the mod. Please update to Release 6

**Comment by AielOlos** [comment]
Yes, It happens to me. I follow a youtube link: https://github.com/LukeRoss00/gta5-real-mod/releases/tag/r4
There I cant't see anything about versions and I did think it was last one, after a lot of time, I saw:  https://github.com/LukeRoss00/gta5-real-mod/releases and fps problem go away. The problem of links in internet are when you click, can't see easy that there are more versions. 
PD: Sorry for my bad english :)
